app/build.gradle

```diff
@@ -234,6 +234,7 @@ dependencies {
     implementation 'com.android.support:preference-v7:27.1.1'
     implementation 'com.jakewharton:process-phoenix:2.0.0'
     debugImplementation 'com.facebook.stetho:stetho:1.5.0'
+    implementation "org.mozilla.components:fretboard:0.16"
```

Comment:
Above is a block of other components that uses "$mozilla_components_version" to keep the version number in sync.

Reply:
I'll use it.